internal/cloud/realm/client.go

```diff
@@ -31,6 +31,8 @@ type Client interface {
 	DeleteApp(groupID, appID string) error
 	FindApps(filter AppFilter) ([]App, error)
 
+	FindSecrets(groupID, appID string) ([]Secret, error)
```

Comment:
I usually reserve the `Find` word when a call has some form of options/filter to it, and generally prefixing a method with `Get` is not very go-like...  let's go with `ListSecrets` here

Reply:
Changed to `Secrets` as discussed offline